app/gateway-oss/2.5.x/hybrid-mode.md

```diff
@@ -417,38 +417,50 @@ automatically within seconds.
 
 ### Version compatibility
 
-{{site.ce_product_name}} control planes only connect to data planes with the
-same major version and at most two minor versions earlier (edge included).
-Control planes won't connect to data planes with newer versions.
+{{site.ce_product_name}} control planes only allow connections from data planes
+with the same major version.
+Control planes won't allow connections from data planes with newer minor versions.
 
 For example, a {{site.ce_product_name}} v2.4.2 control plane:
```

Comment:
I think maybe we want to "rebase" the version examples to 2.5 here. Since although the
logic is correct, in realty a 2.4.2 Kong won't have those behaviour though.

Reply:
Changed.